functions/Write-DbaDbTableData.ps1

```diff
@@ -108,9 +113,9 @@ function Write-DbaDbTableData {
 
     .EXAMPLE
         PS C:\> $tableName = "MyTestData"
-        PS C:\> $query = "SELECT name, create_date, owner_sid FROM sys.databases"
-        PS C:\> $dataset = Invoke-DbaQuery -SqlInstance 'localhost,1417' -SqlCredential $containerCred -Database master -Query $query
-        PS C:\> $dataset | Select-Object name, create_date, @{L="owner_sid";E={$_."owner_sid"}} | Write-DbaDbTableData -SqlInstance 'localhost,1417' -SqlCredential $containerCred -Database tempdb -Table myTestData -Schema dbo -AutoCreateTable
+        PS C:\> $query = "SELECT name, create_date FROM sys.databases"
+        PS C:\> $dataset = Invoke-DbaQuery -SqlInstance 'localhost,1417' -SqlCredential $containerCred -Database master -Query $query -As DataSet
+        PS C:\> $dataset | Write-DbaDbTableData -SqlInstance 'localhost,1417' -SqlCredential $containerCred -Database tempdb -Table $tableName -AutoCreateTable
```

Comment:
This would be a good example of something to add to the post/wiki page: how to deal with binary data.

Reply:
Should work now, please have a look at the commits of today.